libraries/testtags/src/main/kotlin/io/element/android/libraries/testtags/Compose.kt

```diff
@@ -26,9 +26,7 @@ import androidx.compose.ui.semantics.testTagsAsResourceId
  * Add a testTag to a Modifier, to be used by external tool, like TrafficLight for instance.
  */
 @OptIn(ExperimentalComposeUiApi::class)
-fun Modifier.testTag(id: TestTag) = this.then(
-    semantics {
-        testTag = id.value
-        testTagsAsResourceId = true
-    }
-)
+fun Modifier.testTag(id: TestTag) = semantics {
+    testTag = id.value
+    testTagsAsResourceId = true
+}
```

Comment:
Could also have been fixed by:

```kotlin
fun Modifier.testTag(id: TestTag) = this.then(
    Modifier.semantics { // <- Note the addition here
        testTag = id.value
        testTagsAsResourceId = true	
    }
)
```

Reusing the same modifier is explaining why the padding was applied several times.